client/my-sites/email/titan-add-mailboxes/titan-mailbox-pricing-notice.jsx

```diff
@@ -23,6 +23,66 @@ const doesAdditionalPriceMatchStandardPrice = ( domain, titanMonthlyProduct ) =>
 	);
 };
 
+function getPriceMessage( purchaseCost, expiryDate, translate ) {
```

Comment:
Nit: it may be easier in the long run to use a single object argument from which we extract the data we need.
```suggestion
function getPriceMessage( { expiryDate, purchaseCost, translate } ) {
```